src/Components/Footer/Footer.jsx

```diff
@@ -0,0 +1,42 @@
+import "./footer.css";
+
+export const Footer = () => {
+  const toTop = "#";
+  return (
+    <>
+      <a href={toTop} className="btn btn-float-action" id="back-to-top">
+        <i className="fas fa-arrow-up"></i>
+      </a>
```

Comment:
Avoid using <a> tag, you can use Link tag here

Reply:
This vanilla js way to go to top. I will check if there is any other react way to go to top of the page.